src/api/routes/subscriptions.js

```diff
@@ -348,6 +353,7 @@ subscriptions.addSubscription = function (app, res, applications, loggedInUserId
                     clientSecret: clientSecret,
                     auth: selectedApi.auth,
                     approved: !needsApproval,
+                    customId : customIdData,
```

Comment:
remove this too
add this snippet after newSusbcription

  
                if (subsCreateInfo.api === "cortellies-api-collection"){
                    newSubscription.customId = userInfo.customId;
                }